services/wikipedia-service/src/index.ts

```diff
@@ -1,34 +1,32 @@
-import initAspecto from "@aspecto/opentelemetry";
+import initAspecto from '@aspecto/opentelemetry';
 initAspecto({
   aspectoAuth:
-    process.env.ASPECTO_AUTH ?? "e97d7a26-db48-4afd-bba2-be4d453047eb",
-  local: process.env.NODE_ENV !== 'production',
+    process.env.ASPECTO_AUTH ?? 'e97d7a26-db48-4afd-bba2-be4d453047eb',
   logger: console,
   packageName: `wikipedia-service(${process.env.MODE.toLowerCase()})`,
```

Comment:
`packageName` is deprecated. we now call it `serviceName`.
I suggest to fix it also if you're up for it